mslice/cli/__init__.py

```diff
@@ -1,2 +1,38 @@
-from ._mslice_commands import * # noqa: F401
-from mslice.plotting.pyplot import * # noqa: F401
+from __future__ import (absolute_import, division, print_function)
+
+import mantid.kernel
+
+from matplotlib.axes import Axes
+from matplotlib.projections import register_projection
+from mslice.cli.cli_helperfunctions import is_slice, is_cut
+
+
+# MSlice Matplotlib Projection
+class MSliceAxes(Axes):
+    name = 'mslice'
+
+    def plot(self, *args, **kwargs):
+        from mslice.cli.cli_mslice_projection_functions import PlotCutMsliceProjection
+        if is_cut(*args):
+            try:
+                lines = PlotCutMsliceProjection(*args, **kwargs)
+                return lines
+            except Exception as e:
```

Comment:
Do we need to catch exceptions here? I think it would be better to let them out.